_docs/flavour/routing.md

```diff
@@ -2,9 +2,11 @@
 title: Routing
 ---
 
-# Creating route interface
+Routing is a powerful Flavour feature that lets you easily create single-page web applications.
```

Comment:
Please, remove this. Flavour itself is a framework that lets you create SPA. You can re-phrase this section on your will, but you have to describe what's routing actually for. BTW, the first paragraph of 'creating a route interface' contains the description.

Reply:
Routing is so powerful and important in Flavour.  I wanted to emphasize it somehow.  But I agree, I will remove this sentence.